FirstClasses/multiples/src/multiples.erl

```diff
@@ -1,5 +1,39 @@
 -module(multiples).
 
 -export([thousand_sum/0]).
+-export([generate_input_collection/1]).
```

Comment:
Unless you tested it, or use it in a different file, there is no need for exporting this function ;)

Reply:
yes, sorry i forgot to delete this line. It left from dev stage :)